### PR TITLE
Fix iOS layout issue

### PR DESCRIPTION
**Pull request by vicnicius**
Currently, there's an issue when rendering the social messenger demo on IOS, due to how the view height is calculated.

This fixes the issue by setting the height of the app container dynamically.

Before (notice the app nav bar missing on the top):

Screenshot 2021-05-18 at 12 43 57

After:

Screenshot 2021-05-18 at 12 43 05

Ref:
- https://stream-io.atlassian.net/browse/WEF-477
- https://stream-io.atlassian.net/browse/CRS-571
- https://dev.to/maciejtrzcinski/100vh-problem-with-ios-safari-3ge9

### CODE REVIEW

File: social-messenger/src/App.css [hunk 1]
```diff
@@ -1,9 +1,13 @@
+:root {
+  --app-height: 100%;
+}
 html,
 body {
   margin: 0;
   padding: 0;
   height: 100%;
   background: #212326;
+  overflow: hidden;
```
Comment (vicnicius) [inline social-messenger/src/App.css:10]: 🔪

File: social-messenger/src/App.js [hunk 2]
```diff
@@ -80,6 +80,20 @@ const App = () => {
...
+    /*
+     * Get the actual rendered window height to set the container size properly
+     * as in some browsers (like iOS) the browser nav bar might override
+     * the app.
+     */
+    const appHeight = () => {
+      const doc = document.documentElement;
+      doc.style.setProperty('--app-height', `${window.innerHeight}px`);
+    };
+    window.addEventListener('resize', appHeight);
```
Comment (tatianainama) [inline social-messenger/src/App.js:93]: It would be nice to remove the listener on component unmount

**Comment by jaapbakker88** [comment]
@DanC5 @ambaldwin I havent looked at the TS version of the demo yet, but could you make sure this change is also made on that end example

File: social-messenger/src/App.js (hunk 2 above)
Comment (vicnicius) [reply social-messenger/src/App.js:93]: Hecho! https://github.com/GetStream/website-react-examples/pull/23/commits/2816313c4e9702fad2d44e26c153d5c80b86468d

**Review by DanC5 — APPROVED** [review]

**Comment by DanC5** [comment]
> @DanC5 @ambaldwin I havent looked at the TS version of the demo yet(sorry about that), but could you make sure this change is also made on that end example

All good, making those TS updates now then will build and push up to the branch so it will all be updated when you're ready to switch the pointer to the TS social app

**Pull request merged**